src/class-parsely.php

```diff
@@ -77,7 +77,7 @@ class Parsely {
 		'logo'                      => '',
 		'metadata_secret'           => '',
 		'disable_autotrack'         => false,
-		'plugin_version'            => '',
+		'plugin_version'            => self::VERSION,
```

Comment:
I think we should add test for this case and make sure that this case will not break in future also.

Reply:
Merging as-is, but added a note about it in #1161 so we can do that in the future.